src/server/cmd/job-shim/main.go

```diff
@@ -45,16 +45,16 @@ func main() {
 	cmdutil.Main(do, &appEnv{})
 }
 
-func downloadInput(c *client.APIClient, commitMounts []*fuse.CommitMount) error {
+func downloadInput(puller *sync.Puller, c *client.APIClient, commitMounts []*fuse.CommitMount) error {
```

Comment:
Pretty sure we don't need any code related to `job-shim` anymore.  Looks like it might've been merged into `pfs-v2` mistakenly earlier.  Could you remove it?